sotopia/experimental/agents/moderator.py

```diff
@@ -294,17 +349,286 @@ async def aeval(self, epilog: EpisodeLog) -> EpisodeLog:
         print("episode eval finished")
         return epilog
 
+    async def setup_groups(self, groups_data: dict) -> None:
```

Comment:
let's not do the group here, moderator suppose to be simple and general, if we want a specific channel, we should start another moderator